tests/Aspire.Hosting.Redis.Tests/Aspire.Hosting.Redis.Tests.csproj

```diff
@@ -15,7 +15,11 @@
 
   <ItemGroup>
     <Compile Include="$(RepoRoot)src\Aspire.Hosting.Redis\RedisContainerImageTags.cs" />
-    <Compile Include="$(RepoRoot)src\Aspire.Hosting.Redis\RedisCommanderConfigWriterHook.cs" />
+    <Compile Remove="C:\Code\aspire\src\Aspire.Hosting.Redis\RedisCommanderConfigWriterHook.cs" />
+    <Compile Remove="C:\Code\aspire\src\Aspire.Hosting.Redis\RedisCommanderConfigWriterHook.cs" />
+    <Compile Remove="C:\Code\aspire\src\Aspire.Hosting.Redis\RedisCommanderConfigWriterHook.cs" />
+    <Compile Remove="C:\Code\aspire\src\Aspire.Hosting.Redis\RedisCommanderConfigWriterHook.cs" />
+    <Compile Remove="C:\Code\aspire\src\Aspire.Hosting.Redis\RedisCommanderConfigWriterHook.cs" />
```

Comment:
Remove
```suggestion
```

Reply:
what the heck ... how did they get in there?